### ISSUE TITLE
Not support bazelisk

### ISSUE THREAD

**Issue by wingdi** [bazelbuild/intellij#3365]
My computer installed bazelisk,  when i import bazel project to Android Studio, the command default use bazel , not bazelisk, will you add bazelisk support ？

**Comment by AlexeyGy** [comment]
I am not familiar with it but it seems that [bazelisk](https://github.com/bazelbuild/bazelisk) promises to be a fully transparent wrapper. Hence, you could try setting the Bazel binary location as "bazelisk" under Settings > Other Settings > Bazel settings. Does that work?

**Comment by wingdi** [comment]
Very thanks for your quick reply.

I changed to use bazel , 

There seems no way changing bazel to bazelisk. 
<img width="970" alt="截屏2022-03-22 15 31 54" src="https://user-images.githubusercontent.com/25787391/159431586-552ef214-e2fc-40b6-89df-bdf88a005f44.png">


I mean when import bazel project, i want change  `bazel info ... ` to `bazelisk info ... `, or when click build button, it invoke `bazelisk build ` rather than `bazel build`

**Comment by AlexeyGy** [comment]
Are you not able to change the input? If so, this must be a bug in the UI. This is how it looks for me:
<img width="739" alt="image" src="https://user-images.githubusercontent.com/13885630/159435686-83cbc6ff-0b24-4ad5-a788-8835a6ad78ee.png">

**Comment by wingdi** [comment]
You are right, after changing bazel to bazelisk, the command bacome bazelisk instead of bazel.
